bindings/node.js/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "c-kzg",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
Found the issue after running the publish command with 4.0.0 so needed to bump the minor.  Not sure why this showed up now though.